docs/how_to_make_a_tahoe-lafs_release.org

```diff
@@ -1,80 +1,96 @@
+  - [ ] release version is reporting itself as intended version
+  - [ ] created tarballs (for later comparision)
+  - [ ] 'git pull' doesn't pull anything
   - [ ] git tag -s -u 68666A7A -m "release Tahoe-LAFS-X.Y.Z" tahoe-lafs-X.Y.Z
-    - produces a "signed tag"
-  - [ ] build locally to make sure the release is reporting itself as the
-    intended version, make tarballs too (for comparison only)
-  - [ ] push tag to trigger buildslaves. Making a code change is no longer
-    necessary (TODO: pushing just 1.12.0b2 was insufficient: travis fired,
-    but not buildbot)
-    - git push official master TAGNAME
-    - that will build tarballs
-  - [ ] make sure buildbot is green
-  - [ ] download tarballs+wheels
-  - [ ] announce tree is unlocked
-  - [ ] compare tarballs+wheels against local copies (but sign upstreams)
-  - [ ] sign each with "gpg -ba -u 68666a7a FILE"
-  - [ ] twine upload dist/*
-  - [ ] test "pip install tahoe-lafs" (from PyPI)
-  - [ ] upload *.asc to org ~source/downloads/
-  - [ ] copy the release tarball,sigs to tahoe-lafs.org: ~source/downloads/
-  - [ ] move old release out of ~source/downloads (to downloads/old/?)
-  - [ ] send out relnotes.txt:
-    - add prefix with SHA256 of tarballs, release pubkey, git revhash
-    - GPG-sign the email with release key
-    - send to tahoe-announce@tahoe-lafs.org and tahoe-dev@tahoe-lafs.org
-  - [ ] update Wiki front page: version on download link, News column
-  - [ ] update Wiki "Doc": parade of release notes (with rev of NEWS.rst)
-  - [ ] close the Milestone on the trac Roadmap
-  - [ ] tweet to @tahoelafs
-  - other stuff:
-  - [ ] update https://tahoe-lafs.org/hacktahoelafs/
-  - [ ] make an "announcement of new release" on freshmeat
+     XXX should use full key-id above
+  - [ ] pushed tag to trigger buildslaves: git push official master TAGNAME
+
+* sign the release artifacts [0/8]
+
+  - [ ] pushed signed tag (should trigger Buildbot builders)
+  - [ ] Buildbot workers built all artifacts successfully
+  - [ ] downloaded upstream tarballs+wheels
+  - [ ] announce on IRC that master is unlocked
+  - [ ] compared upstream tarballs+wheels against local copies
+  - [ ] signed each upstream artifacts with "gpg -ba -u 68666a7a FILE"
+     XXX should use full key-id above
+  - [ ] added to relnotes.txt: [0/3]
+    - [ ] prefix with SHA256 of tarballs
+    - [ ] release pubkey
+    - [ ] git revision hash
+  - [ ] GPG-signed the release email with release key (write to relnotes.txt.asc)
+
+* publish release artifacts [0/9]
+
+  - [ ] uploaded to PyPI via: twine upload dist/*
+  - [ ] uploaded *.asc to org ~source/downloads/
+  - [ ] test install works properly: pip install tahoe-lafs
+  - [ ] copied the release tarballs and signatures to tahoe-lafs.org: ~source/downloads/
+  - [ ] moved old release out of ~source/downloads (to downloads/old/?)
+  - [ ] updated readthedocs.org (XXX what does this mean? it updates automatically, no?)
```

Comment:
It seems to.  In case not, or if there is other configuration to be managed, perhaps @warner can give someone access to the tahoe-lafs project on readthedocs.  Happy for that someone to be me (and I can grant others access as necessary/appropriate).  My readthedocs username is jcalderone